tickets.md

```diff
@@ -34,14 +34,18 @@ The training day is paid for separately (and includes lunch on Thursday).
 
 ### What are the COVID mitigations?
 
-COVID-19 still causes illness and long-term harm across Europe. We want to keep attendees safe, while at the same time allowing for as much familiarity and community spirit as is possible. For this reason our policy at the conference will be:
+COVID-19 still causes illness and long-term harm across the world ([source](https://www.who.int/news/item/03-12-2025-WHO-launches-new-unified-plan-for-countries-to-manage-coronaviruses-COVID-19-and-beyond)). 
 
-<ul>
-<li>masks will be provided and should be worn in all indoor conference spaces (except while eating/drinking)</li>
-<li>lateral flow tests will be provided for each day (conference organisers should be notified if the test is positive)</li>
-</ul>
+We want to keep attendees safe, while at the same time allowing for as much familiarity and community spirit as is possible. <br>
+Each measure on its own has flaws but together offer better protection.
 
-(For the next edition (2026), we hope to sort out active air filtering so we can make masks recommended instead of mandatory.)
+For this reason our policy at the conference this year will be:
+
+* masks will be provided and we strongly recommend wearing them in all indoor conference spaces (except while eating/drinking)
+* lateral flow tests will be provided for each day and mandatory to perform (conference organisers should be notified if the test is positive)
+* air filtering will be provided in meeting rooms that don't have windows to the outside
+
+We encourage you to test if possible before you start travelling. We had one person arrive with covid in 2025 and will explain shortly what we did and what we can expect from each other if that happens again in 2026.
```

Comment:
By shortly, I mean in the next week or so hopefully, but it feels we could go live without it?